install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -0,0 +1,136 @@
+= Backing Docker Registry with GlusterFS Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+== Overview
+
+This guide will cover the how to attach a GlusterFS persistent volume to the Docker Registry.
+It is assumed that the docker registry service has already been started and the gluster volume has been created.
+
+
+=== Prerequisites
+* The link:../install/docker_registry.html#deploy-registry[docker-registry] has been deployed *without*
+configuring storage.
+* A gluster volume exists and *glusterfs-fuse* is installed on schedulable nodes.
+* Definitions written for GlusterFS link:../persistent_storage/persistent_storage_glusterfs.html#creating-gluster-endpoints[endpoints and service],
+link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[persistent volume (*pv*)],
+and
+link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claim (*pvc*)].
+** For this guide, these will be:
+*** `gluster-endpoints-service.yaml`
+*** `gluster-endpoints.yaml`
+*** `gluster-pv.yaml`
+*** `gluster-pvc.yaml`
+* A user with the
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[*cluster-admin*] role binding.
+** For this guide, that user is `admin`
+
+NOTE:  All `$ oc` commands are executed on the master node as the `admin` user
+
+
+[[create-gfs-pvc]]
+=== Create the Gluster Persistent Volume
+
+First, make the Gluster volume available to the registry.
+
+----
+$ oc create -f gluster-endpoints-service.yaml
+$ oc create -f gluster-endpoints.yaml
+$ oc create -f gluster-pv.yaml
+$ oc create -f gluster-pvc.yaml
+----
```

Comment:
Do you want to mention `oc get pv; oc get pvc` as a way to verify that the pv and pvc bound properly?